### PR TITLE
Update 13 - Signature Activation Protocol.md

### PR DESCRIPTION
**Pull request by Razumain**
Resolves issue #188

### CODE REVIEW

File: 13 - Signature Activation Protocol.md [hunk 1]
```diff
@@ -426,3 +428,7 @@ Protection profile for QSCD for Server Signing
 
 - The `RequestedVersion` element of the `SADRequestType` is now marked as optional in the schema definition.
 
+**Changes between version 1.1 and 1.2:**
+
+- The use of "signmessage" LoA was removed.
```
Comment (martin-lindstrom) [inline 13 - Signature Activation Protocol.md:433]: "sign message"

File: 13 - Signature Activation Protocol.md [hunk 2]
```diff
@@ -426,3 +428,7 @@ Protection profile for QSCD for Server Signing
...
+- The protocol logic was clarified by removing reference to sign message (which was never present in the actual protocol) and replacing this with a clarification of the semantics of including the the Sign Request ID in SAD to assert the acceptance to sign the documents referenced in that Sign Request.
```
Comment (martin-lindstrom) [inline 13 - Signature Activation Protocol.md:434]: "by removing reference**s** to sign message ..."

**Review by martin-lindstrom — CHANGES_REQUESTED** [review]
Some small stuff. Also update versions.md.

File: 13 - Signature Activation Protocol.md [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 # Signature Activation Protocol for Federated Signing
 
-### Version 1.1 - 2020-01-17
+### Version 1.2 - 2022-10-18
```
[the remark this answers is not in the document]
Comment (Razumain) [reply 13 - Signature Activation Protocol.md]: Fixat

File: 13 - Signature Activation Protocol.md (hunk 2 above)
Comment (Razumain) [reply 13 - Signature Activation Protocol.md]: I have rearranged and clarified. "signmessage" in quotes refers to the actual extension of the LoA URI:s that have been deprecated.

**Review by martin-lindstrom — CHANGES_REQUESTED** [review]
The versions.md is still not updated.

Also. When adding issues, also add the issue to the correct project (Post Nov 2021). This helps us to see what has been changed between versions.

**Review by martin-lindstrom — APPROVED** [review]

**Pull request merged**